### PR TITLE
Add LICENSE, README and Contributing

### PR DESCRIPTION
**Pull request by DunderRoffe**
Add basic documentation to the repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+META-PELUX-RPI-BSP
```
Comment (GordanM) [inline README.md]: This should be meta-pelux-bsp-rpi if I am not mistaken.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+====================
+Yocto layer adding configurations required when building PELUX towards raspberry pi.
```
Comment (GordanM) [inline README.md]: Not sure what are the capitalization rules but would prefer "Raspberry Pi" instead of all small caps.

Comment (erikboto) [reply README.md]: I agree, let's try to use Raspberry Pi

File: README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Maintained at https://github.com/pelagicore/meta-pelux-bsp-rpi
```
Comment (erikboto) [inline README.md]: There should be a capital P in Pelagicore: https://github.com/Pelagicore/meta-pelux-bsp-rpi

**Comment by erikboto** [comment]
Thanks for adding this, was on my TODO!

**Comment by DunderRoffe** [comment]
Good catches. Think I fixed all of them.
@erikboto no problem :)

**Review by GordanM — APPROVED** [review]

**Review by erikboto — APPROVED** [review]

**Pull request merged**